lib/ansible/modules/storage/purestorage/purefa_hg.py

```diff
@@ -84,16 +68,16 @@
     api_token: e31060a7-21fc-e277-6240-25983c6c4592
 '''
 
-RETURN = '''
+RETURN = r'''
```

Comment:
It's to make this a raw string. It may not be necessary in this case but it is for other modules, so I always do it by default.